### PR TITLE
[12.0] account_invoice_import backport hooks from v13

### PR DESCRIPTION
**Pull request by yankinmax**
Backport
- [ ] https://github.com/OCA/edi/pull/213

### CODE REVIEW

**Review by grindtildeath — COMMENTED** [review]
LG, but would be better to have it split in two commits as is the case with:
 - https://github.com/OCA/edi/pull/213/commits/d0ab4986aee8da2869291290d31e8daebcfe5df3
 - https://github.com/OCA/edi/pull/213/commits/b492501185e007246778f518f70e4b12ebfc0e96

**Review by i-vyshnevska — COMMENTED** [review]
can you cherry-pick commits pointed by Akim to keep the initial author?

File: account_invoice_import/wizard/account_invoice_import.py [hunk 1]
```diff
@@ -1005,8 +1022,7 @@ def message_new(self, msg_dict, custom_values=None):
                 logger.info(
                     'Attachment %d: %s. Trying to import it as an invoice',
                     i, attach.fname)
-                parsed_inv = self.parse_invoice(
-                    base64.b64encode(attach.content), attach.fname)
```
Comment (i-vyshnevska) [inline account_invoice_import/wizard/account_invoice_import.py]: this change doesn't look LG, here work is happening on multiple attachments relying on file on wizard is wrong

**Review by yvaucher — APPROVED** [review]

**Comment by yvaucher** [comment]
Code review, LGTM

(commits to squash)

**Review by i-vyshnevska — APPROVED** [review]

**Comment by yvaucher** [comment]
@yankinmax I want to retry the merge. Sounds like it was due to some runbot error.

Can you rebase this?

File: account_invoice_import/wizard/account_invoice_import.py [hunk 2]
```diff
@@ -481,51 +481,41 @@ def invoice_already_exists(self, commercial_partner, parsed_inv):
...
-    def _get_missing_config(self, partner, company):
-        configs = partner.invoice_import_ids.filtered(
-            lambda r: r.company_id == company)
-        if not configs:
-            raise UserError(_(
-                "Missing Invoice Import Configuration on partner '%s'."
-            ) % partner.display_name)
-        return configs[0].convert_to_import_config()
-
-    def _get_child_company(self, company_id, parsed_inv):
```
Comment (etobella) [inline account_invoice_import/wizard/account_invoice_import.py]: Why are you removing this?

Comment (yvaucher) [reply account_invoice_import/wizard/account_invoice_import.py]: Looks suspicious indeed, probably a bad conflict resolution.

Comment (yankinmax) [reply account_invoice_import/wizard/account_invoice_import.py]: Yeah, I see. Sorry for that, will rebase correctly.

Comment (yankinmax) [reply account_invoice_import/wizard/account_invoice_import.py]: Is it good now?

**Review by etobella — APPROVED** [review]

**Pull request merged**